docs/src/prototypes/meetingOptions/components/MSTeamsDivider.tsx

```diff
@@ -0,0 +1,10 @@
+import { Divider as StardustUIDivider } from '@stardust-ui/react'
+
+export default props => {
```

Comment:
would rather name it as `InvisibleDivider` - this seems to better reflect the idea behind it

Reply:
Yep, make sense.